src/Symfony/Component/Mime/Part/DataPart.php

```diff
@@ -156,6 +156,9 @@ public function __wakeup()
             throw new \BadMethodCallException('Cannot unserialize '.__CLASS__);
         }
         foreach (['body', 'charset', 'subtype', 'disposition', 'name', 'encoding'] as $name) {
+            if ('body' === $name && isset($this->_parent[$name]) && $this->_parent[$name] instanceof File) {
+                $this->_parent[$name] = file_get_contents($this->_parent[$name]->getPath());
```

Comment:
This is the same logic as how this would usually happen in `\Symfony\Component\Mime\Part\TextPart::getBody` if the email had not been serialized.